client/components/language-picker/language-names.js

```diff
@@ -0,0 +1,141 @@
+	'az': translate => translate( 'Azerbaijani' ),
+	'ba': translate => translate( 'Bashkir' ),
+	'bel': translate => translate( 'Belarusian' ),
+	'bg': translate => translate( 'Bulgarian' ),
+	'bm': translate => translate( 'Bambara' ),
+	'bn': translate => translate( 'Bangla' ),
+	'bo': translate => translate( 'Tibetan' ),
+	'br': translate => translate( 'Breton' ),
+	'bs': translate => translate( 'Bosnian' ),
+	'ca': translate => translate( 'Catalan' ),
+	'ce': translate => translate( 'Chechen' ),
+	'cs': translate => translate( 'Czech' ),
+	'csb': translate => translate( 'Kashubian' ),
+	'cv': translate => translate( 'Chuvash' ),
+	'cy': translate => translate( 'Welsh' ),
+	'da': translate => translate( 'Danish' ),
+	'de': translate => translate( 'German' ),
+	'dv': translate => translate( 'Divehi' ),
+	'dz': translate => translate( 'Dzongkha' ),
+	'el': translate => translate( 'Greek' ),
+	'el-po': translate => translate( 'Polytonic Greek' ),
+	'en': translate => translate( 'English' ),
+	'en-gb': translate => translate( 'British English' ),
+	'eo': translate => translate( 'Esperanto' ),
+	'es': translate => translate( 'Spanish' ),
+	'es-cl': translate => translate( 'Chilean Spanish' ),
+	'et': translate => translate( 'Estonian' ),
+	'eu': translate => translate( 'Basque' ),
+	'fa': translate => translate( 'Persian' ),
+	'fi': translate => translate( 'Finnish' ),
+	'fo': translate => translate( 'Faroese' ),
+	'fr': translate => translate( 'French' ),
+	'fr-be': translate => translate( 'Belgian French' ),
+	'fr-ca': translate => translate( 'Canadian French' ),
+	'fr-ch': translate => translate( 'Swiss French' ),
+	'fur': translate => translate( 'Friulian' ),
+	'fy': translate => translate( 'Western Frisian' ),
+	'ga': translate => translate( 'Irish' ),
+	'gd': translate => translate( 'Scottish Gaelic' ),
+	'gl': translate => translate( 'Galician' ),
+	'gn': translate => translate( 'Guarani' ),
+	'gu': translate => translate( 'Gujarati' ),
+	'he': translate => translate( 'Hebrew' ),
+	'hi': translate => translate( 'Hindi' ),
+	'hr': translate => translate( 'Croatian' ),
+	'hu': translate => translate( 'Hungarian' ),
+	'hy': translate => translate( 'Armenian' ),
+	'ia': translate => translate( 'Interlingua' ),
+	'id': translate => translate( 'Indonesian' ),
+	'ii': translate => translate( 'Sichuan Yi' ),
+	'ilo': translate => translate( 'Iloko' ),
+	'is': translate => translate( 'Icelandic' ),
+	'it': translate => translate( 'Italian' ),
+	'ja': translate => translate( 'Japanese' ),
+	'ka': translate => translate( 'Georgian' ),
+	'kir': translate => translate( 'Kyrgyz' ),
+	'kk': translate => translate( 'Kazakh' ),
+	'km': translate => translate( 'Khmer' ),
+	'kn': translate => translate( 'Kannada' ),
+	'ko': translate => translate( 'Korean' ),
+	'ks': translate => translate( 'Kashmiri' ),
+	'ckb': translate => translate( 'Central Kurdish' ),
+	'kv': translate => translate( 'Komi' ),
+	'la': translate => translate( 'Latin' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **10** times:
`translate( 'Latina' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).